src/vs/workbench/parts/terminal/electron-browser/terminalPanel.ts

```diff
@@ -149,25 +149,30 @@ export class TerminalPanel extends Panel {
 				// occurs on the selection itself.
 				this._terminalService.getActiveInstance().focus();
 			} else if (event.which === 3) {
-				// Trigger the context menu on right click
-				let anchor: HTMLElement | { x: number, y: number } = this._parentDomElement;
-				if (event instanceof MouseEvent) {
```

Comment:
I just realized that this is actually my code haha, as long as the context menu appears in the correct spot after this change (when `rightClickCopyPaste` is false) then this was fine to remove :+1: